apps/wallet-dashboard/components/migration/migration-object-details-card/MigrationObjectDetailsCard.tsx

```diff
@@ -115,41 +114,30 @@ interface UnlockConditionLabelProps {
     isTimelocked: boolean;
 }
 function UnlockConditionLabel({ groupKey, isTimelocked: isTimelocked }: UnlockConditionLabelProps) {
-    const { data: currentEpochStartTimestampMs, isLoading: isLoadingEpochStart } =
-        useGetCurrentEpochStartTimestamp();
     const { data: currentEpochEndTimestampMs, isLoading: isLoadingEpochEnd } =
         useGetCurrentEpochEndTimestamp();
 
     const epochEndMs = currentEpochEndTimestampMs ?? 0;
-    const epochStartMs = currentEpochStartTimestampMs ?? '0';
-    const currentDateMs = Date.now();
-
     const unlockConditionTimestampMs = parseInt(groupKey) * MILLISECONDS_PER_SECOND;
-    const isFromPreviousEpoch =
-        !isLoadingEpochStart && unlockConditionTimestampMs < parseInt(epochStartMs);
     // TODO: https://github.com/iotaledger/iota/issues/4369
     const isInAFutureEpoch = !isLoadingEpochEnd && unlockConditionTimestampMs > epochEndMs;
-
     const outputTimestampMs = isInAFutureEpoch ? unlockConditionTimestampMs : epochEndMs;
 
-    const timeLabel = useTimeAgo({
-        timeFrom: outputTimestampMs,
-        shortedTimeLabel: true,
-        shouldEnd: true,
-        maxTimeUnit: TimeUnit.ONE_DAY,
+    const formattedLastPayoutExpirationTime = useCountdownByTimestamp(Number(outputTimestampMs), {
+        showSeconds: false,
     });
-
-    const showLabel = !isFromPreviousEpoch && outputTimestampMs > currentDateMs;
```

Comment:
This change makes the logic incorrect, because:
`const outputTimestampMs = isInAFutureEpoch ? unlockConditionTimestampMs : epochEndMs;`
if the unlockConditionTimestampMs is in the past (a past epoch possibly) the timestamp used is the **current epoch timestamp end**, so even expired objects will show a expiration time, when really, they should have expired already

Reply:
This is related to `https://github.com/iotaledger/iota/issues/4369`, where the object could have expired today, but won't really expire until the end of the epoch, that's why that one is used

Reply:
paused until the rest of the team comes back, and we can discuss it :wink:

Reply:
The logic for not showing a label in previous epochs was fine as it was before